Packs/Devo/Integrations/Devo_v2/Devo_v2.py

```diff
@@ -823,13 +824,42 @@ def multi_table_query_command(offset, items):
     return entry
 
 
+def convert_to_str(value):
+    if isinstance(value, list) and len(value) == 0:
+        return_warning("Empty list encountered.")
+        return '[]'
+    elif isinstance(value, dict) and not value:
+        return_warning("Empty dictionary encountered.")
+        return '{}'
+    elif isinstance(value, list | dict):
+        return json.dumps(value)
+    return str(value)
+
+
 def write_to_table_command():
-    table_name = demisto.args()["tableName"]
-    records = check_type(demisto.args()["records"], list)
+    tag = demisto.args().get("tag", None)
+    tableName = demisto.args()["tableName"]
+    records_str = demisto.args()["records"]
```

Comment:
You can use argToList function cause this records argument IsArray base on the .yml file
this function converts a string representation of args to a python list

and then you can remove the parsing and the check if the records its a list

Reply:
Actually the records list array consists of every datatypes i.e. list, boolean, integer, etc. So were directly fetching the data from records parameter. Else if we implement your suggestion, the XSOAR platform gives an error like this, "Failed to execute command devo-write-to-table. Error: the JSON object must be str, bytes or bytearray, not list."

Reply:
You need to add the records without enclosing brackets ([]),  as a list of strings separated by commas.